apps/payments/next/app/[locale]/[offeringId]/[interval]/location/page.tsx

```diff
@@ -2,24 +2,82 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import * as Sentry from '@sentry/nextjs';
+import { headers } from 'next/headers';
+import Image from 'next/image';
+import { notFound, redirect } from 'next/navigation';
+
 import {
+  Banner,
+  BannerVariant,
   BaseParams,
   IsolatedSelectTaxLocation,
   buildRedirectUrl,
 } from '@fxa/payments/ui';
-import { config } from 'apps/payments/next/config';
 import { fetchCMSData } from '@fxa/payments/ui/actions';
 import { getApp, TermsAndPrivacy } from '@fxa/payments/ui/server';
-import { headers } from 'next/headers';
-import { notFound, redirect } from 'next/navigation';
-import Image from 'next/image';
 import locationIcon from '@fxa/shared/assets/images/confirm-pairing.svg';
-import infoIcon from '@fxa/shared/assets/images/icon_information_circle_outline_current.min.svg';
-import * as Sentry from '@sentry/nextjs';
 import type { PageContentOfferingTransformed } from '@fxa/shared/cms';
+import { LocalizerRsc } from '@fxa/shared/l10n/server';
+import { config } from 'apps/payments/next/config';
 
 export const dynamic = 'force-dynamic';
 
+const LocationBanner = ({
```

Comment:
Instead of a separate component, could this just be included in the location page?

Reply:
Yep, I think this would be better in the location page once the logic has been moved. 👍🏼